.github/workflows/ci.yml

```diff
@@ -73,17 +75,17 @@ jobs:
       - run: poe lint
       - run: poe build-develop
       - run: mkdir junit-xml
-      - run: poe test ${{matrix.pytestExtraArgs}} -s --junit-xml=junit-xml/${{ matrix.python }}--${{ matrix.os }}.xml
+      - run: poe test ${{matrix.pytestExtraArgs}} -s --junit-xml=junit-xml/${{ matrix.python }}--${{ matrix.os }}--${{ matrix.dependencies }}.xml
```

Comment:
Let's remove references to `matrix.dependencies` in these filenames (here and 4 more spots below)